app/main.py

```
@@ -1,17 +1,21 @@
 from fastapi import FastAPI
-
+from app.logger import setup_logging, get_logger
 
 @asynccontextmanager
 async def lifespan(app: FastAPI):
+    setup_logging()
+    logger = get_logger("app")
+    logger.info("Application startup: Logging configured")
     try:
         await redis_client.connect()
     except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)